Resources/config/plugins.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <service id="httplug.plugin.cache" class="Http\Client\Plugin\CachePlugin" public="false">
+            <argument />
```

Comment:
Hm, no it is not completed. I missed that. I have excluded a couple of plugin that require dependencies. The cache plugin must have a PSR6 cache pool and there is no way for me to give a good default. 

I think the best solution is to allow some sort of configuration like: 

```yaml
httplug: 
  plugins: 
    cache:
      service: 'acme.cache'
      config: 
        respect_headers: true
    logger:
      formatter: 'acme.formatter'
```

If you don't mind, could I do that in a separate PR?